src/components/v2v3/shared/NftVideo/JuiceVideoThumbnailOrImage.tsx

```diff
@@ -0,0 +1,58 @@
+import { LoadingOutlined } from '@ant-design/icons'
+import { ImageProps } from 'antd'
+import { useContentType } from 'hooks/ContentType'
+import { useState } from 'react'
+import { classNames } from 'utils/classNames'
+import { MP4_FILE_TYPE } from '../FundingCycleConfigurationDrawers/NftDrawer/NftUpload'
+import { JuiceVideoThumbnail, PlayIconPosition } from './JuiceVideoThumbnail'
+
+export function JuiceVideoThumbnailOrImage({
+  isSelected,
+  playIconPosition,
+  ...props
+}: ImageProps & {
+  isSelected?: boolean
+  playIconPosition?: PlayIconPosition
+  src: string
```

Comment:
**question** is `src` not already in ImageProps?

Reply:
Can be undefined in ImageProps is the only reason I included here